phpunit/functional/DocumentTest.php

```diff
@@ -859,32 +901,32 @@ public function testCronCleanorphans()
             'entities_id'     => 0,
             '_documents_id'   => [$did3]
         ]);
-        $this->integer((int)$tickets_id_1)->isGreaterThan(0);
-        $this->boolean($ticket->getFromDB($tickets_id_1))->isTrue();
+        $this->assertGreaterThan(0, (int)$tickets_id_1);
+        $this->assertTrue($ticket->getFromDB($tickets_id_1));
 
         $docitem = new \Document_Item();
-        $this->boolean($docitem->getFromDBByCrit(['itemtype' => 'Ticket', 'items_id' => $tickets_id_1]))->isTrue();
+        $this->assertTrue($docitem->getFromDBByCrit(['itemtype' => 'Ticket', 'items_id' => $tickets_id_1]));
 
        // launch Cron for closing tickets
         $mode = - \CronTask::MODE_EXTERNAL; // force
         \CronTask::launch($mode, 5, 'cleanorphans');
 
        // check documents presence
-        $this->boolean($doc->getFromDB($did1))->isFalse();
-        $this->boolean($doc->getFromDB($did2))->isFalse();
-        $this->boolean($doc->getFromDB($did3))->isTrue();
+        $this->assertFalse($doc->getFromDB($did1));
+        $this->assertFalse($doc->getFromDB($did2));
+        $this->assertTrue($doc->getFromDB($did3));
     }
 
     public function testGetDuplicateOf()
     {
-        $instance = $this->newTestedInstance();
+        $instance = new \Document();
 
        // Test when the file is not in the DB
-        $output = $instance->getDuplicateOf(0, __DIR__ . '/../fixtures/uploads/foo.png');
-        $this->boolean($output)->isFalse();
+        $output = $instance->getDuplicateOf(0, __DIR__ . '/../../tests/fixtures/uploads/foo.png');
```

Comment:
In main I've added a constant for the fixture dir:

![image](https://github.com/user-attachments/assets/420758a2-d0d6-41b7-b883-7a7d9465e50e)

Maybe we should add it in bugfix and use it too ?
The idea is that when we move the `phpunit` folder back into `test`, we can just change the constant and be done with it.

Reply:
Doen in latest commit